src/order/order-request-body.ts

```diff
@@ -39,5 +39,5 @@ export interface OrderPaymentRequestBody {
      * An object that contains the details of a credit card or vaulted payment
      * instrument.
      */
-    paymentData?: CreditCardInstrument | VaultedInstrument;
+    paymentData?: PaymentInstrument;
```

Comment:
Can you not export `PaymentInstrument`? `PaymentInstrument` is a union type that combines `CreditCardInstrument | NonceInstrument | VaultedInstrument | CryptogramInstrument`. Only `CreditCardInstrument | VaultedInstrument` should be made public. Others should be hidden from the client because they are internal details (i.e.: clients cannot pass `nonce` as part of `OrderRequestBody` because it is an internal detail).

Reply:
👍